### DISCUSSION TITLE
Examples you would like to see

### DISCUSSION THREAD

**Discussion by dstroy0** [dstroy0/InputHandler#55] [General]
Any ideas for examples you would like, you can post here and we'll give it a whirl!

**Comment by 2bndy5** [comment]
having an example that uses a different stream is very useful. I like the current examples which cover Serial and MQTT streams. 

I think the GPS parser idea could be considered an advanced example.

**Comment by dstroy0** [comment]
RF24, lora, zigbee.. any others?

**Comment by dstroy0** [comment]
I think there's telnet across most platforms too so I can see if that is feasible

**Comment by 2bndy5** [comment]
BLE UART

**Comment by dstroy0** [comment]
the teensy 4.1 has PHY ethernet and a good amount of resources, I wonder if that one can do SSH, I'll look into arduino/esp SSH while I'm looking into telnet

**Comment by dstroy0** [comment]
I'm planning on starting on this once the userInput ctor overhaul is complete and wildcards have been implemented, by then the api should hopefully not be going through any more growing pains and it will make sense to work on the examples

**Comment by dstroy0** [comment]
Just pushed an RF24 example, it's a repurposed GettingStarted

**Comment by dstroy0** [comment]
Pushed LoRa tx/rx

**Comment by dstroy0** [comment]
pushed ble uart

**Comment by dstroy0** [comment]
I think I'm going to use TelnetStream as the Telnet lib.  https://github.com/JAndrassy/TelnetStream and we can add TLS with this lib: https://github.com/OPEnSLab-OSU/SSLClient

**Comment by dstroy0** [comment]
debugging example is up

**Comment by dstroy0** [comment]
I made a config_howto example and focused the debugging example some more

**Comment by dstroy0** [comment]
I think I'm going to leave out telnet.
